smart-contract/src/state.rs

```diff
@@ -6,6 +6,7 @@ use cosmwasm_storage::{
     bucket, bucket_read, singleton, singleton_read, Bucket, ReadonlyBucket, ReadonlySingleton,
     Singleton,
 };
+use crate::msg::VoteChoice;
```

Comment:
not sure if its best practice to pull in enum from another file vs defining it again and mapping between enums 🤷

Reply:
The import is fine.